rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -304,7 +304,7 @@ case object LongComparator extends CriterionType {
 
 case object MemoryComparator extends CriterionType {
   override val comparators = OrderedComparators.comparators
-  override protected def validateSubCase(v:String,comparator:CriterionComparator) =  try {
+  override protected def validateSubCase(v:String,comparator:CriterionComparator) = {
```

Comment:
because it is not supposed to throw exception (Full/Failure), the try must be a remeniscience of the past